From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Mon, 20 Jan 2020 15:07:50 +0000
Subject: xml validity

Avoid appstream XML file validation failure, by using git tag -l
instead of git tag --list.  Could also git tag -""-list, yuck.

$ lintian
E: xournalpp: appstream-metadata-invalid com.github.xournalpp.xournalpp.appdata.xml

$ appstreamcli validate desktop/com.github.xournalpp.xournalpp.appdata.xml
E: ~:~: xml-markup-invalid Could not parse XML data: Entity: line 80: parser error : Double hyphen within comment: <!-- Could be automated more:
	 `git tag 
	 `git tag --list | grep "^[0-9]*\.[0-9]*\.[0-9]*$`"
	          ^
Validation failed: errors: 1
---
 desktop/com.github.xournalpp.xournalpp.appdata.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/desktop/com.github.xournalpp.xournalpp.appdata.xml b/desktop/com.github.xournalpp.xournalpp.appdata.xml
index f9fc26c..62dc9e6 100644
--- a/desktop/com.github.xournalpp.xournalpp.appdata.xml
+++ b/desktop/com.github.xournalpp.xournalpp.appdata.xml
@@ -77,7 +77,7 @@
   <content_rating type="oars-1.0"/>
   <releases>
     <!-- Could be automated more:
-	 `git tag --list | grep "^[0-9]*\.[0-9]*\.[0-9]*$`"
+	 `git tag -l | grep "^[0-9]*\.[0-9]*\.[0-9]*$`"
     -->
     <release date="2019-11-10" version="1.0.16"/>
     <release date="2019-10-15" version="1.0.15"/>
